docs/subcommands/encoding.md

```diff
@@ -1,10 +1,10 @@
 # encoding
 
-The `encoding` subcommand detects the likely PHRED score encoding scheme used for FASTQ files. PHRED scores are encoded as ASCII characters, but which ASCII characters encode for which relative quality scores depends on the build and generation of the sequencing machine used. `ngsderive` obtained details of the different encoding schemes [here][https://en.wikipedia.org/wiki/FASTQ_format#Encoding].
+The `encoding` subcommand detects the likely PHRED score encoding scheme used for FASTQ and BAM files. PHRED scores are encoded as ASCII characters, but which ASCII characters encode for which relative quality scores depends on the build and generation of the sequencing machine used. The BAM specification calls for `PHRED+33` (AKA Illumina1.8+ or Sanger encoding), however that is also the most permissive encoding. It is possible for a stricter encoding to be mis-translated as `PHRED+33` and appear to be of higher quality than it is in truth. This subcommand can be used to check if base scores are suspiciously high throughout a BAM, and a mis-translation may have occured.
```

Comment:
```suggestion
The `encoding` subcommand detects the likely PHRED score encoding scheme used for FASTQ and BAM files. PHRED scores are encoded as ASCII characters, but which ASCII characters encode for which relative quality scores depends on the build and generation of the sequencing machine used. The `encoding` subcommand supports PHRED+33 (AKA Illumina1.8+ or Sanger), Illumina1.0, and Illumina1.3 quality encoding formats. The BAM specification calls for `PHRED+33`, however that is also the most permissive encoding. It is possible for a stricter encoding to be mis-translated as `PHRED+33` and appear to be of higher quality than it is in truth. This is due to Illumina1.3 using ASCII characters that are a subset of those in Illumina 1.0 and Illumina 1.0 using a subset of characters in Sanger format (that is Illumina1.3 &#8834; Illumina1.0 and Illumina1.0 &#8834; Sanger. This subcommand can be used to check if base scores are suspiciously high throughout a BAM, and a mis-translation may have occurred.
```

Reply:
You may also want to provide some description or example of what you mean by checking for mis-translation. Essentially, this depends on low-quality bases to detect the encoding. If the data are high quality, this will choose a more restrictive encoding, which may not accurately reflect the original choice by the data source.